sklearn/datasets/tests/test_svmlight_format.py

```diff
@@ -5,6 +5,7 @@
 import scipy.sparse as sp
 import os
 import shutil
+from importlib import resources as importlib_resources
```

Comment:
I think we could just use:

```suggestion
from importlib import resources
```

and then call `resources.open_binary` / `open_binary.open_text` instead of renaming this module every where.